CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/ReleaseCockpitGenerator.kt

```diff
@@ -28,38 +28,17 @@ internal class ReleaseCockpitGenerator : BaseCockpitGenerator() {
         return createGetterMethodSpecForParamAndConfigurator(param) { builder ->
             val value = param.value
             when (value) {
-                is List<*> -> builder.addStatement(createArrayListStatement(value as ArrayList<*>).toString(), arraysClassName)
+                is List<*> -> builder.addStatement(createListStatement(value as List<Any>), arraysClassName)
                 else -> builder.addStatement("return ${createWrappedValueForParamValue(param.value)}")
             }
         }
     }
 
-    private fun createArrayListStatement(value: ArrayList<*>): StringBuilder {
-        val stringBuilder = StringBuilder()
-        stringBuilder.append("return new ArrayList<>(\$T.asList(")
-
-        val itemSeparator = ", "
-        value.forEach {
-            it?.let {
-                stringBuilder.append(createWrappedValueForParamValue(it))
-            } ?: stringBuilder.append("null")
-            stringBuilder.append(itemSeparator)
-        }
-        val lastIndexOf = stringBuilder.lastIndexOf(itemSeparator)
-        if (lastIndexOf != -1) {
-            stringBuilder.delete(lastIndexOf, lastIndexOf + itemSeparator.length) // remove comma and space after last value
-        }
-        stringBuilder.append("))")
-
-        return stringBuilder
-    }
+    private fun createListStatement(value: List<Any>) = value.map { createWrappedValueForParamValue(it) }
+            .joinToString(separator = ", ", prefix = "return new List<>(\$T.asList(", postfix = "))")
```

Comment:
Method seems to be unused, because of no getters for `list` type. Otherwise `List` class should be parametrized with `\$T`. Btw, we can't instantiate interface `List`, should be rather `ArrayList`